server/sonar-server/src/test/java/org/sonar/server/qualitygate/RegisterQualityGatesTest.java

```diff
@@ -52,32 +63,165 @@
 
   @Rule
   public DbTester db = DbTester.create(System2.INSTANCE);
+  @Rule
+  public LogTester logTester = new LogTester();
+
+  private static final String A_RATING_VALUE = Integer.toString(RatingGrid.Rating.A.getIndex());
```

Comment:
Why not `private static final String A_RATING_VALUE = "A"` ?

Reply:
or `RatingGrid.Rating.A.name()` ?

Reply:
I did not dig further I just copy this from existing code.